### DISCUSSION TITLE
Badge Support for Repositories

### DISCUSSION THREAD

**Discussion by diogormendes** [aquasecurity/trivy#4416] [Ideas]
Is anything in the Roadmap to have badges for the repository like Snyk does ?

Reference: [LINK](https://support.snyk.io/hc/en-us/articles/360003997277-Badge-Support-for-Repositories)

**Comment by ghost** [comment]
agreed, this would be a nice feature.

**Comment by AnaisUrlichs** [comment]
hmm we could consider this -- I just want to add that Snyk is a semi-open source (more a free) tool. They integrate much more with their enterprise offering than Trivy does. So I am just curious if people would want to use it & you would need to be able to configure it to the type of severity you would like to show etc.

I think this would be a great idea for a GH Action that is community maintained. Just my two cents

**Comment by comminutus** [comment]
I'd like to see this as well!

**Comment by itaysk** [comment]
@diogormendes @comminutus can you please elaborate on how you expect this to work? where do you perform your scan and how will the badge know the result

**Reply by comminutus** [reply]
@itaysk I perform the scan using Github Actions.  It'd be neat to generate a badge icon in the scan that I can then use in the readme, similar to other lint/build/code quality tools' badges.

**Reply by itaysk** [reply]
I still don't see how we can technically support this, considering that trivy Trivy is just a CLI tool and has no service/api. In your case, since you are using Trivy in GH Actions, perhaps using the GH Actions Workflow badge will help you.

**Comment by diogormendes** [comment]
@itaysk , same way as described here: https://support.snyk.io/hc/en-us/articles/360003997277-Badge-Support-for-Repositories

**Reply by itaysk** [reply]
Trivy is an open source CLI tool. Snyk is a SaaS service with an API. So the comparison isn't helpful. If you are using Trivy in GH actions, perhaps this approach will help you: https://github.com/aquasecurity/trivy/discussions/4416#discussioncomment-9604078
